src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +64,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
+        if (isset($config['persistent']) && $config['persistent']) {
+            throw new \InvalidArgumentException('Sqlserver driver does not allow config setting: persistent');
```

Comment:
Why don't we use a message that actually explains the problem, like ``Sqlserver does not support perstistent connections set through config settings persistent``. Or even better we state: ``The SQL Server PHP driver does not support persistent connections. Please set config setting "persistent" to "false".``.